stable/cockroachdb/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: cockroachdb
 home: https://www.cockroachlabs.com
-version: 3.0.7
-appVersion: 19.2.5
+version: 3.0.8
```

```suggestion
version: 4.0.0
```
it is a major bump in the app version, need reflect in the chart version

also if users upgrade from 19.xxx to 20.xx there are any breaking changes?

---

There are some backwards-incompatible changes to some of the SQL functions documented in the release notes here: https://www.cockroachlabs.com/docs/releases/v20.1.0.html#backward-incompatible-changes -- is there somewhere I should be linking that?

---

Please link in the Readme and make a section to describe how to upgrade existing deployments to this new release